components/chart/tooltip/shared.md

```diff
@@ -0,0 +1,146 @@
+
+* [Parameter Settings](#parameter-settings) - lets you alter cosmetic settings such as borders, colors and padding through simple parameters
+* [Shared Template](#shared-template) - lets you control the entire content
+
+### Parameter Settings
+You can customize the rendering of the `Shared` tooltip by using:
+
+* `Background` - control the background color by applying a CSS color string, including HEX and RGB. By default the it will match the color for the category.
+* `Color` - control the text color by applying a CSS color string, including HEX and RGB.
+* `Opacity` - control the opacity of the tooltip.
+* `Border` - control the `Color` and the `Width` of the tooltip by using the `<ChartSeriesTooltipBorder />` nested inside the `<ChartSeriesTooltip>` tag.
+* `Padding` - control the `Left`, `Right`, `Top` and `Bottom` padding of the tooltip by using the `<ChartSeriesTooltipPadding />` nested inside the `<ChartSeriesTooltip>` tag.
+
+
+### Shared Template
+
+The `SharedTemplate` allows you to control the rendering of the shared tooltip.
+
+It provides `context` which contains the following information:
+* `Category` - renders the name of the Category.
+* `Point` - provides the following information on the points:
+
+@[template](/_contentTemplates/chart/link-to-basics.md#context-parameter-information)
+
+
+>caption Usage of the SharedTemplate
+
+````CSHTML
+@* This example shows how to use the SharedTemplate and extract information on the data points value and category from the context *@
+
+<TelerikChart>
+    <ChartTooltip Visible="true" Shared="true">
+        <SharedTemplate>
+           @{
+               var points = context.Points;
+
+               foreach (var point in points)
+               {
+                   <div>
+                       <TelerikIcon Icon="@IconName.Information" />
+                       Point value: @point.Value
+                       Category: @context.Category
+                   </div>
+               }
+           }
+        </SharedTemplate>
+    </ChartTooltip>
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Bar" Name="Product 1" Data="@series1Data">
+        </ChartSeries>
+        <ChartSeries Type="ChartSeriesType.Bar" Name="Product 2" Data="@series2Data">
+        </ChartSeries>
+    </ChartSeriesItems>
+
+    <ChartCategoryAxes>
+        <ChartCategoryAxis Categories="@xAxisItems"></ChartCategoryAxis>
+    </ChartCategoryAxes>
+
+    <ChartTitle Text="Quarterly revenue per product"></ChartTitle>
+
+    <ChartLegend Position="ChartLegendPosition.Right">
+    </ChartLegend>
+</TelerikChart>
+
+@code {
+    public List<object> series1Data = new List<object>() { 10, 2, 5, 6 };
+    public List<object> series2Data = new List<object>() { 5, 8, 2, 7 };
+    public string[] xAxisItems = new string[] { "Q1", "Q2", "Q3", "Q4" };
+}
+````
+>caption The result from the code snippet above
+
+![shared tooltip example](images/shared-tooltip-template.png)
```

Comment:
Could you take the screenshot again but with a smaller browser. The entire image could be about 600px wide, or even a bit less so it does not become blurry and constantly resize. Also, this will reduce its size.